src/test/playwright/e2e/exercise/modeling/ModelingExerciseAssessment.spec.ts

```
@@ -44,6 +44,9 @@ test.describe('Modeling Exercise Assessment', () => {
         // });
 
         test('Tutor can assess a submission', async ({ login, courseManagement, courseAssessment, exerciseAssessment, modelingExerciseAssessment }) => {
+            await login(tutor, '/course-management');
+            await courseManagement.openSubmissionsForExerciseAndCourse(course.id!, modelingExercise.id!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)